doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -50,7 +48,7 @@ There are no entries for this section yet.
 MCUboot
 =======
 
-The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``7a51968``, plus some |NCS| specific additions.
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``680ed07``, plus some |NCS| specific additions.
```

Comment:
Why is the SHA changing here? after the release we keep having the same version of MCUBoot?

Reply:
This changed at the last minute before the release (we updated the SHA together with you and @SebastianBoe ), so I'm mirroring what is in the 1.7.0 release notes.